sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -266,127 +223,21 @@ scatter(Tx *p, Toffset offset, simd<Tx, N> vals, simd_mask<N> mask = 1) {
   scatter<Tx, N>(p, simd<Toffset, N>(offset), vals, mask);
 }
 
-/// Loads a contiguous block of memory from the given memory address \p addr
-/// and returns the loaded data as a vector.
-/// The generated code depends on the combination {T, N, Flags}.
-/// Providing flags specifying the alignment of 16-bytes or more produces more
-/// efficient code. If the alignment is smaller than 16-bytes, then less
-/// efficient gather is generated. If the loaded vector is too long
-/// for 1 flat-load GPU instruction, then a series of flat-loads and/or gathers
-/// may be generated.
-/// @tparam Tx Element type.
-/// @tparam N Number of elements to load.
-/// @tparam Flags The alignment specifier type tag.
-/// @param addr The address to load from.
-/// @param Flags Specifies the alignment.
-/// @return A vector of loaded elements.
-///
-template <typename Tx, int N,
-          typename Flags = overaligned_tag<detail::OperandSize::OWORD>>
-__ESIMD_API std::enable_if_t<is_simd_flag_type_v<Flags>, simd<Tx, N>>
-block_load(const Tx *addr, Flags) {
```

Comment:
This function and the next 2 are not removed - they are just moved, grouped with other user-visible block_load() functions (and block_store() functions).